packages/modules/vehicles/smarteq/config.py

```diff
@@ -1,17 +1,26 @@
 from typing import Optional
+import json
```

Comment:
Wenn Du aus dem Paket `json` nur `json.dumps()` verwendest, macht es dann nicht Sinn, auch nur das zu importieren?

Reply:
Es werden noch ein paar (wenige) andere benutzt (dump, load, ...)

Reply:
Aber doch nicht in dieser Datei, oder ich bin blind.

Reply:
Stimmt, ich habe in api.py nachgesehen, sorry. 
Ist geändert.